modules/manually-create-iam-ibm-cloud.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations.adoc
+
+[id="manually-create-iam-ibm-cloud_{context}"]
+= Manually creating IAM for IBM Cloud
+
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets for you cloud provider.
+
+You can use the Cloud Credential Operator (CCO) utility (`ccoctl`) to create the required IBM Cloud resources.
+
+.Prerequisites
+
+* You have configured the `ccoctl` binary.
+* You have an existing `install-config.yaml` file.
+
+.Procedure
+
+. Edit the `install-config.yaml` configuration file so that it contains the `credentialsMode` parameter set to `Manual`.
++
+.Example `install-config.yaml` configuration file
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Manual <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Manual`.
+
+. To generate the manifests, run the following command from the directory that contains the installation program:
++
+[source,terminal]
+----
+$ openshift-install create manifests --dir <installation_directory>
+----
+
+. From the directory that contains the installation program, obtain details of the {product-title} release image that your `openshift-install` binary is built to use:
++
+[source,terminal]
+----
+$ openshift-install version
+----
++
+.Example output
+[source,terminal]
+----
+release image quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64
+----
+
+. Extract the `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal]
+----
+$ oc adm release extract --cloud=ibmcloud --credentials-requests $RELEASE_IMAGE \
+    --to=<path_to_directory_with_list_of_credentials_requests> <1>
```

Comment:
It's not clear to me what the path_to_directory_with_list_of_credentials_requests is.  Is that the 'manifests' directory created in https://github.com/openshift/openshift-docs/pull/39767/files#diff-9df224f76a24729d514ee8a469d3a87a0022dd03942afda1136e7090280e525eR38 ?  If so, I think it would be helpful to call that out explicitly.